schwifty/bank_registry/manual_it.json

```diff
@@ -1,666 +1,1122 @@
+		"primary": true,
+		"short_name": "CASSA RAIFFEISEN VAL BADIA"
+	},
+	{
+		"bank_code": "08011",
+		"bic": "CCRTIT2T01A",
+		"country_code": "IT",
+		"name": "CASSA RURALE VALLAGARINA - BANCA DI CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA RURALE VALLAGARINA"
+	},
+	{
+		"bank_code": "08016",
+		"bic": "CCRTIT2T04A",
+		"country_code": "IT",
+		"name": "CASSA RURALE ALTOGARDA - ROVERETO",
+		"primary": true,
+		"short_name": "CASSA RURALE ALTOGARDA - ROVERETO"
+	},
+	{
+		"bank_code": "08078",
+		"bic": "CCRTIT2T20A",
+		"country_code": "IT",
+		"name": "LA CASSA RURALE - CC ADAMELLO GIUDICARIE VALSABBIA PAGANELLA - SC",
+		"primary": true,
+		"short_name": "LA CASSA RURALE"
+	},
+	{
+		"bank_code": "08102",
+		"bic": "CCRTIT2T27A",
+		"country_code": "IT",
+		"name": "CASSA RURALE VALSUGANA E TESINO - BANCA DI CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA RURALE VALSUGANA E TESINO"
+	},
+	{
+		"bank_code": "08178",
+		"bic": "CCRTIT2T47A",
+		"country_code": "IT",
+		"name": "CASSA RURALE ALTA VALSUGANA - BANCA DI CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA RURALE ALTA VALSUGANA"
+	},
+	{
+		"bank_code": "08183",
+		"bic": "RZSBIT21027",
+		"country_code": "IT",
+		"name": "CASSA RAIFFEISEN PRATO-TUBRE SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA RAIFFEISEN PRATO"
+	},
+	{
+		"bank_code": "08234",
+		"bic": "RZSBIT21033",
+		"country_code": "IT",
+		"name": "CASSA RAIFFEISEN DI SCENA SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA RAIFFEISEN DI SCENA SOCIETA"
+	},
+	{
+		"bank_code": "08304",
+		"bic": "CCRTIT2T76A",
+		"country_code": "IT",
+		"name": "Cassa Rurale di Trento Banca di Credito Cooperativo",
+		"primary": true,
+		"short_name": ""
+	},
+	{
+		"bank_code": "08324",
+		"bic": "ICRAITRR900",
+		"country_code": "IT",
+		"name": "BANCA CENTROPADANA CREDITO COOPERATIVO - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA CENTROPADANA"
+	},
+	{
+		"bank_code": "08327",
+		"bic": "ICRAITRRROM",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI ROMA SOC.COOP. A R.L.",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO DI ROMA"
+	},
+	{
+		"bank_code": "08329",
+		"bic": "ICRAITRR950",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO BRIANZA E LAGHI - SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO COOPERATIVO BRIANZA E LAGHI"
+	},
+	{
+		"bank_code": "08340",
+		"bic": "CCRTIT2TPAD",
+		"country_code": "IT",
+		"name": "CASSA PADANA BANCA DI CREDITO COOPERATIVO SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "CASSA PADANA BANCA"
+	},
+	{
+		"bank_code": "08342",
+		"bic": "CCRTIT2TAQU",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO COOPERATIVO DI AQUARA SOCIETA' COOPERATIVA",
+		"primary": true,
+		"short_name": "BCC AQUARA"
```

Comment:
[low]: do you think we should keep `DI AQUARA` in the `short_name`?

Reply:
on its website it was shown as BCC AQUARA